### PR TITLE
DOC-7375: Add YouTube include to render embedded videos

### PR DESCRIPTION
**Pull request by nickvigilante**
Fixes DOC-7375

VS Code also added the missing header links for us in the Style Guide.

### CODE REVIEW

File: StyleGuide.md [hunk 1]
```diff
@@ -1070,6 +1120,26 @@ Use the following HTML and Liquid to include an image in a Markdown page:
 
 Example: [Decommission Nodes](https://www.cockroachlabs.com/docs/stable/remove-nodes.html#step-1-check-the-node-before-decommissioning)
 
+<a name="videos"></a>
+
+### Videos
+
+Like images, use videos to clarify a topic, but only use them as needed. Typically, videos should be hosted on the official [CockroachDB YouTube page](https://www.youtube.com/@cockroachdb).
```
Comment (lnhsingh) [inline StyleGuide.md:1127]: Should we add something here about videos usually being surfaced by Marketing?

File: StyleGuide.md [hunk 2]
```diff
@@ -1070,6 +1120,26 @@ Use the following HTML and Liquid to include an image in a Markdown page:
...
+
+Use the following Liquid to include an embedded video in a Markdown page:
+
+~~~ md
+{% include_cached youtube.html video_id="<YouTube ID>" [widescreen=true] %}
+~~~
+
+The `video_id` parameter is required. The `widescreen` parameter is optional and meant for videos with wide banners such as our video on [Foreign Key Constraints](https://www.youtube.com/watch?v=5kiMg7GXAsY).
```
Comment (lnhsingh) [inline StyleGuide.md:1135]: Maybe clarify that the video_id is the value after v= in the URL?

**Review by lnhsingh — APPROVED** [review]
A couple clarifications needed in the Style Guide, but otherwise LGTM. Thanks for making this update!!

File: StyleGuide.md (hunk 1 above)
Comment (nickvigilante) [reply StyleGuide.md]: Done!

File: StyleGuide.md (hunk 2 above)
Comment (nickvigilante) [reply StyleGuide.md]: Done!

**Pull request merged**